torch/csrc/jit/passes/utils/memory_dag.cpp

```diff
@@ -31,12 +32,65 @@ bool MemoryDAG::mayAliasImpl(const Element* a, const Element* b) const {
   return false;
 }
 
+bool MemoryDAG::mayContainAlias(
```

Comment:
Let's make the argument names more consistent (here and in the header file). In particular, I think "contained"/"container" and "elem/container" are misleading with the new commutative implementation.